packages/react-composites/tests/app/lib/MockCallAdapter.ts

```diff
@@ -152,7 +152,24 @@ export class MockCallAdapter implements CallAdapter {
   }
 
   blurVideoBackground(): Promise<void> {
-    throw new Error('blurVideoBackground not implemented.');
+    this.modifyState((draft: CallAdapterState) => {
+      if (!draft.call) {
```

Comment:
if there's no call, this should set the unparented video stream of the devicemanager

Reply:
done